src/main/java/net/sf/jabref/external/DroppedFileHandler.java

```diff
@@ -560,7 +555,7 @@ private boolean doCopy(String fileName, ExternalFileType fileType, String toFile
         if (found < 0) {
             // OOps, we don't know which directory to put it in, or the given
             // dir doesn't exist....
-            System.out.println("dir: " + dirs.get(found) + "\t ext: " + fileType.getExtension());
+            // This should not happen!!
```

Comment:
You're probably correct. I just copied the "The this should not happen from the corresponding place in `doMove`.